infoschema/sysschema/const.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2016 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sysschema
+
+var sysSchemaTables = []string{
+	viewUnusedIndexes,
+	viewIndexUsage,
+}
+
+const viewUnusedIndexes = "CREATE VIEW sys." + viewNameUnusedIndexes +
+	` AS SELECT DISTINCT i.table_schema AS table_schema, i.table_name AS table_name, i.key_name AS index_name
```

Comment:
Why use `DISTINCT` in these two SQLs?

Reply:
Consider a situation， table `t` has an index idx_ab(a, b). There will be two rows of data in the table `information_schema.tidb_indexes` because this index covers two columns. So there will be two lines of the same (table_schema, t, idx_ab). `DISTINCT` is to deal with this situation.